src/pages/assistant/index.tsx

```diff
@@ -51,6 +52,17 @@ export const getServerSideProps = withGlobalData(
         };
       }
 
+      const potential = getAssistantTripIfCached(tripQuery);
```

Comment:
Should we also just add http cache headers here? (potentially as a separate PR)

Reply:
Sure, I'll look into it